README.md

```diff
@@ -75,7 +75,8 @@ development and community news:
 - Follow us on [Twitter] and [Facebook].
 - Subscribe to the [Mixxx Development Blog][blog].
 - Join the developer [mailing list].
-- Post on the [Mixxx forums][forums].
+- Post on the [Mixxx discord channel][discord].
```

Comment:
Discord and Discourse are two different things. We don't have discord.

Reply:
This should be solved by now after rebasing.